### DISCUSSION TITLE
Filter out words like um, ah, eh / transcription

### DISCUSSION THREAD

**Discussion by DannyDevSe** [WyattBlue/auto-editor#544] [2. Feature Ideas]
A feature where it would remove unwanted words would save even more time

**Comment by sasa** [comment]
I second that. Although this is probably something that an AI library could/should do that would be chained into the processing. I often record with a lot of pauses which get thrown out by AE. Another idea would be a keyword or sound which if detected would delete everything up until the second to last pause. That would give an automatic cut not only in the case of um, ah, eh, but also if you just fumble everything you just said which also happens very often (at least to me).

So: Bla (is good audio), comma is pause, Error (is an unwanted piece of speech), BEEP is the correction word/sound

Bla, Bla, Bla, Error, BEEP, Bla, Bla, Bla
becomes
Bla, Bla, Bla, Bla, Bla, Bla

This is quite similar to when you use a transscription software like Dragon Naturally Speaking (which must have a hard time with the new AI advances). You dictate and when you make an error you just say “scratch that” and resume.

Just a dream of mine for my workflow. Don't know if anyone else has this problem.

Greetings
Saša
